### PR TITLE
docs: fix links

### LINKED ISSUE #1580: docs: Accessibility page - fix links
https://style.monday.com/?path=/docs/foundations-accessibility--page

image

- [ ] Fix link size
- [ ] Fix link spacing

image

- [ ] Fix link size
- [ ] Fix link spacing

### PR DESCRIPTION
**Pull request by Dhoni77**
Fixes #1580.

- Fix spacing.
- Fix size.

Attachments:

accessibility

figma

### CODE REVIEW

File: src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx [hunk 1]
```diff
@@ -36,9 +36,9 @@ Web Accessibility is based on guidelines published by The World Wide Web Consort
 [WCAG Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize#principle1)
 
 <Tip title="Get our Accessibility checklist">
-  <Link href={accessibilityCheckList} target="_blank">
+  <Link href={accessibilityCheckList} target="_blank" size={Link.sizes.SMALL} withoutSpacing={false}>
```
Comment (SergeyRoyt) [inline src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx:39]:
```suggestion
  <Link href={accessibilityCheckList} size={Link.sizes.SMALL}>
```

File: src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx [hunk 2]
```diff
@@ -79,14 +79,19 @@ The visual presentation of the following have a contrast ratio of at least 3:1 a
   <img src={colorContrastImage} alt="example for non text color contrast" />
 </Frame>
 
-UI Components - Visual information required to identify active UI components and states (including borders for input elements (text input, radio buttons, checkboxes, etc.) as well as Focus Indicator (visible focus around active elements when they receive keyboard focus)<br/>
+UI Components - Visual information required to identify active UI components and states (including borders for input elements (text input, radio buttons, checkboxes, etc.) as well as Focus Indicator (visible focus around active elements when they receive keyboard focus))<br/>
 Graphical Objects - Parts of graphics required to understand the content
 
 <Tip title="Use color contrast tools for code and design">
   To check color contrast in design use{" "}
```
Comment (SergeyRoyt) [inline src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx:86]:
```suggestion
  To check color contrast in design use
```

File: src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx [hunk 3]
```diff
@@ -79,14 +79,19 @@ The visual presentation of the following have a contrast ratio of at least 3:1 a
...
-  <Link href="https://www.figma.com/community/plugin/748533339900865323/Contrast" target="_blank">
-    Contrast figma plugin
-  </Link>{" "}
+  <Link
+    href="https://www.figma.com/community/plugin/748533339900865323/Contrast"
+    target="_blank"
+    size={Link.sizes.SMALL}
+    withoutSpacing={false}
+  >
```
Comment (SergeyRoyt) [inline src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx:92]:
```suggestion
  <Link href="https://www.figma.com/community/plugin/748533339900865323/Contrast" size={Link.sizes.SMALL}>
```

File: src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx [hunk 4]
```diff
@@ -79,14 +79,19 @@ The visual presentation of the following have a contrast ratio of at least 3:1 a
   <img src={colorContrastImage} alt="example for non text color contrast" />
 </Frame>
 
-UI Components - Visual information required to identify active UI components and states (including borders for input elements (text input, radio buttons, checkboxes, etc.) as well as Focus Indicator (visible focus around active elements when they receive keyboard focus)<br/>
+UI Components - Visual information required to identify active UI components and states (including borders for input elements (text input, radio buttons, checkboxes, etc.) as well as Focus Indicator (visible focus around active elements when they receive keyboard focus))<br/>
```
Comment (SergeyRoyt) [inline src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx:82]:
```suggestion
UI Components - Visual information required to identify active UI components and states, including borders for input elements (text input, radio buttons, checkboxes, etc.) as well as Focus Indicator (visible focus around active elements when they receive keyboard focus)<br/>
```

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Left a few notes

**Comment by Dhoni77** [comment]
cc @SergeyRoyt

**Review by SergeyRoyt — APPROVED** [review]
Great, thanks!

**Pull request merged**